CMakeLists.txt

```diff
@@ -104,7 +106,7 @@ endif()
 
 option(PCAPPP_ENABLE_CLANG-TIDY "Run Clang-Tidy static analysis during build" OFF)
 
-if (PCAPPP_ENABLE_CLANG-TIDY)
+if(PCAPPP_ENABLE_CLANG-TIDY)
```

Comment:
nit: I'd rename `PCAPPP_ENABLE_CLANG-TIDY` to `PCAPPP_ENABLE_CLANG_TIDY` (it's less nice mixing `-` and `_`)